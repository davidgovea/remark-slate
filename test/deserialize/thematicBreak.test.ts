import { deserialize } from '../../src';
import thematicBreakast from '../fixtures/thematic-break';

it('When the list has thematic breaks', () => {
  expect(
    thematicBreakast.children.map((k) => deserialize(k))
  ).toMatchSnapshot();
});
