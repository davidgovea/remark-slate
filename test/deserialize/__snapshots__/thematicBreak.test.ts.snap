// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When the list has thematic breaks 1`] = `
Array [
  Object {
    "text": "",
  },
  Object {
    "children": Array [
      Object {
        "text": "foo",
      },
    ],
    "type": "paragraph",
  },
]
`;
